interface/smallTree_HHbtag.h

```diff
@@ -774,78 +780,6 @@ struct smallTree
       m_bH_pt_raw_jetdown.clear() ;
       m_bH_mass_raw_jetup.clear() ;
       m_bH_mass_raw_jetdown.clear() ;
-      m_bjet1_pt_raw_jetup1 = - 1.;
-      m_bjet1_pt_raw_jetup2 = - 1.;
-      m_bjet1_pt_raw_jetup3 = - 1.;
-      m_bjet1_pt_raw_jetup4 = - 1.;
-      m_bjet1_pt_raw_jetup5 = - 1.;
-      m_bjet1_pt_raw_jetup6 = - 1.;
-      m_bjet1_pt_raw_jetup7 = - 1.;
-      m_bjet1_pt_raw_jetup8 = - 1.;
-      m_bjet1_pt_raw_jetup9 = - 1.;
-      m_bjet1_pt_raw_jetup10 = - 1.;
-      m_bjet1_pt_raw_jetup11 = - 1.;
-
-      m_bjet1_pt_raw_jetdown1 = - 1.;
-      m_bjet1_pt_raw_jetdown2 = - 1.;
-      m_bjet1_pt_raw_jetdown3 = - 1.;
-      m_bjet1_pt_raw_jetdown4 = - 1.;
-      m_bjet1_pt_raw_jetdown5 = - 1.;
-      m_bjet1_pt_raw_jetdown6 = - 1.;
-      m_bjet1_pt_raw_jetdown7 = - 1.;
-      m_bjet1_pt_raw_jetdown8 = - 1.;
-      m_bjet1_pt_raw_jetdown9 = - 1.;
-      m_bjet1_pt_raw_jetdown10 = - 1.;
-      m_bjet1_pt_raw_jetdown11 = - 1.;
-
-      m_bjet2_pt_raw_jetup1 = - 1.;
-      m_bjet2_pt_raw_jetup2 = - 1.;
-      m_bjet2_pt_raw_jetup3 = - 1.;
-      m_bjet2_pt_raw_jetup4 = - 1.;
-      m_bjet2_pt_raw_jetup5 = - 1.;
-      m_bjet2_pt_raw_jetup6 = - 1.;
-      m_bjet2_pt_raw_jetup7 = - 1.;
-      m_bjet2_pt_raw_jetup8 = - 1.;
-      m_bjet2_pt_raw_jetup9 = - 1.;
-      m_bjet2_pt_raw_jetup10 = - 1.;
-      m_bjet2_pt_raw_jetup11 = - 1.;
-
-      m_bjet2_pt_raw_jetdown1 = - 1.;
-      m_bjet2_pt_raw_jetdown2 = - 1.;
-      m_bjet2_pt_raw_jetdown3 = - 1.;
-      m_bjet2_pt_raw_jetdown4 = - 1.;
-      m_bjet2_pt_raw_jetdown5 = - 1.;
-      m_bjet2_pt_raw_jetdown6 = - 1.;
-      m_bjet2_pt_raw_jetdown7 = - 1.;
-      m_bjet2_pt_raw_jetdown8 = - 1.;
-      m_bjet2_pt_raw_jetdown9 = - 1.;
-      m_bjet2_pt_raw_jetdown10 = - 1.;
-      m_bjet2_pt_raw_jetdown11 = - 1.;
-
-
-      m_bH_mass_raw_jetup1 = - 1.;
-      m_bH_mass_raw_jetup2 = - 1.;
-      m_bH_mass_raw_jetup3 = - 1.;
-      m_bH_mass_raw_jetup4 = - 1.;
-      m_bH_mass_raw_jetup5 = - 1.;
-      m_bH_mass_raw_jetup6 = - 1.;
-      m_bH_mass_raw_jetup7 = - 1.;
-      m_bH_mass_raw_jetup8 = - 1.;
-      m_bH_mass_raw_jetup9 = - 1.;
-      m_bH_mass_raw_jetup10 = - 1.;
-      m_bH_mass_raw_jetup11 = - 1.;
-
-      m_bH_mass_raw_jetdown1 = - 1.;
-      m_bH_mass_raw_jetdown2 = - 1.;
-      m_bH_mass_raw_jetdown3 = - 1.;
-      m_bH_mass_raw_jetdown4 = - 1.;
-      m_bH_mass_raw_jetdown5 = - 1.;
-      m_bH_mass_raw_jetdown6 = - 1.;
-      m_bH_mass_raw_jetdown7 = - 1.;
-      m_bH_mass_raw_jetdown8 = - 1.;
-      m_bH_mass_raw_jetdown9 = - 1.;
-      m_bH_mass_raw_jetdown10 = - 1.;
-      m_bH_mass_raw_jetdown11 = - 1.;
```

Comment:
why have all these variable been removed?

Reply:
Those variables were duplicated: they exist under the ```bH_mass_raw_jetup/down``` and ```bH_pt_raw_jetup/down``` vectors, which can also be used by the framework which calculates the systematics. I think it s a good principle to try to reduce the size of the skims, which might become a bottleneck.

Reply:
These variables are necessary in the selectionCfg to compute acceptance corrections for the each JES uncertainty so we need to keep them.

Reply:
I've added them back.